README.md

```diff
@@ -539,17 +528,17 @@ The prerequisites are:
 
 * Have a fragment that is active (one that is accepting new nodes)
 
-* Make sure you are in add-node mode (the cursor is shaped like a cross)
+* Make sure you are in add-node mode (the cursor is shaped like a crosshair)
 
 * You need to be working in one of the two top windows in the Data Slices area.
 
 At this point, you only need to choose where to begin, and in which
 direction to go.  If the cursor is near
 an existing node (the node has turned cyan), the auto-segmentation will start
-at that that node.  Otherwise, it will start at the location of the cursor.
+at that node.  Otherwise, it will start at the location of the cursor.
 
-To automatically create new points to the right of the starting point, press
-the right bracket or brace key (`]` or `}`); either key will work.  To create new points to the
+To automatically create new nodes to the right of the first one, press
+the right bracket or brace key (`]` or `}`); either key will work.  To create new nodes to the
```

Comment:
```diff
+ The auto-segmentation algorithm will automatically create new nodes, going as far as [...]

- At this point, you only need to choose where to begin, and in which direction to go. [...]
- Otherwise, it will start at the location of the cursor.
+ You only need to choose where to begin, and in which direction to go. [...]
+ Otherwise, it will create a node at the location of the cursor and continue from there.

- To automatically create new nodes to the right of the first one, press the right bracket or brace key (] or }); either key will work.
- To create new nodes to the left, press the left bracket or brace key ([] or {).
+ To start and head to the right, press the right bracket or brace key (] or }).
+ To start and head to the left, press the left bracket or brace key ([] or {) instead.

- The auto-segmentation algorithm will go as far as [...]
```
I have modified it to explain the process first and then give instructions.
(If this change is okay, will commit manually.)

Reply:
Unresolving comment. Modifications were confirmed to be okay (see https://github.com/KhartesViewer/khartes/pull/17#issuecomment-1793367671)